packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -320,6 +337,30 @@ class _ReorderableListViewState extends State<ReorderableListView> {
         case TargetPlatform.linux:
         case TargetPlatform.windows:
         case TargetPlatform.macOS:
+
+        final ListenableBuilder dragHandle = ListenableBuilder(
+          listenable: _dragIndex,
+          builder: (BuildContext context, Widget? child) {
+            final MouseCursor effectiveMouseCursor = WidgetStateProperty.resolveAs<MouseCursor>(
+              widget.mouseCursor ?? const WidgetStateMouseCursor.fromMap(<WidgetStatesConstraint, MouseCursor>{
+                WidgetState.dragged: SystemMouseCursors.grabbing,
+                WidgetState.hovered: SystemMouseCursors.grab,
+              }),
+              <WidgetState>{
+                if (_dragIndex.value == index)
```

Comment:
Do we really need this? We only need to know whether the widget is being dragged at all, because the mouse will definitely be on the item being dragged.

Reply:
You are right! I made the code simpler in [refactor: Simplify code](https://github.com/flutter/flutter/pull/160246/commits/ce1f0aab1f79608d7f84c4fd4cd78a0662ba1b90)